common/persistence/visibility/store/elasticsearch/query_interceptors.go

```diff
@@ -154,6 +193,16 @@ func (vi *valuesInterceptor) Values(fieldName string, values ...interface{}) ([]
 	return result, nil
 }
 
+func (vi *valuesInterceptor) applyFieldTransformation(transformation fieldTransformation, value interface{}) (interface{}, error) {
+	switch {
+	case transformation.originalField == searchattribute.ScheduleID && transformation.newField == searchattribute.WorkflowID:
+		if strValue, ok := value.(string); ok {
```

Comment:
do we support queries like `WorkflowId IN ('a', 'b', 'c')`? if so, does this support that for `ScheduleId` also?

Reply:
Yes, we do support the `IN` operator.